app/config/dependencies.php

```diff
@@ -75,41 +100,85 @@
     $userRepository,
     $commentRepository,
 );
+$tagService = new TagService(
+    $tagRepository,
+    $ingredientRepository
```

Comment:
So here aswell, using the ingredientService and vice vesa with tagService, but thats also on me.